tests/test.rs

```diff
@@ -46,7 +46,7 @@ mod tests {
                 ]
             }"#;
             let schema: &PyDict = py.eval(code, None, None).unwrap().extract().unwrap();
-            SchemaSerializer::py_new(schema, None).unwrap();
+            SchemaSerializer::py_new(py, schema, None).unwrap();
```

Comment:
You can get rid of this argument I think

Reply:
Compiler yelled at me to add this. I think If you call the Python constructor PyO3 passes it automatically but not when calling the method directly from rust?